Assets/Scripts/PlayerUIManager.cs

```diff
@@ -0,0 +1,60 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+using System.Collections;
+
+namespace Filibusters
+{
+	public class PlayerUIManager : MonoBehaviour 
+	{
+		public CoinInventory mInventoryScript;
+		private Text mCoinText;
+	    private Text mVotesText;
+
+	    void Start()
+	    {
+	    	// Enable the Canvas and assign the main camera to it
+			Camera mainCamera = GameObject.FindGameObjectWithTag("MainCamera").GetComponent<Camera>();
+			var canvas = GetComponent<Canvas>();
+			canvas.enabled = true;
+			canvas.worldCamera = mainCamera;
+
+			// Assign Coin and Votes Text elements
+	        foreach (Transform t in transform)
```

Comment:
Might be a good idea to define a Utility function to do this for us because I ran into something similar in my PR. (not something to do right now, but possibly for the future)?
A function defined like...
GetChildComponentWithTag<T>(GameObject parent, string tag)